CMakeLists.txt

```diff
@@ -137,6 +137,15 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)  # https://cmake.org/cmake/help/
   endif()
 endif()
 
+if(BUILD_JPEG AND WITH_LIBJEG_TURBO_SIMD)
```

Comment:
Need this to set right status for SIMD Support in the line 1353

Reply:
**LGTM** ( i tested only on Windows )

you can see build logs with [nasm is available](https://ci.appveyor.com/project/sturkmen72/opencv-image-reading/builds/44454473?fullLog=true) and [nasm is unavailable](https://ci.appveyor.com/project/sturkmen72/opencv-image-reading/builds/44454275?fullLog=true)

result output of [test code](https://github.com/sturkmen72/opencv-image-reading/blob/jpg/test-2.cpp) is like below

**SIMD Support:            YES**
```
Frames captured (VideoCapture) :   100    Average FPS:     170.8    Average time per frame:      5.85 ms
Frames captured (imread)       :   100    Average FPS:     109.0    Average time per frame:      9.17 ms
```

**SIMD Support:            NO**
```
Frames captured (VideoCapture) :   100    Average FPS:     171.1    Average time per frame:      5.84 ms
Frames captured (imread)       :   100    Average FPS:      82.9    Average time per frame:     12.06 ms
```

Reply:
I realized  i do not need that part. I removed but did not pushed yet. It does not affect anything anyway.